### ISSUE TITLE
WitAi doesn't work

### ISSUE THREAD

**Issue by Rei-x** [Rei-x/discord-speech-recognition#15]
Currently, speech recognition with witAi doesn't work.

**Comment by lmmfranco** [comment]
Yep. Been trying to fiddle with it for some days. (TBH just trying random things since I'm not experienced with audio encoding)
Currently we can reach a point where wit.ai will accept the file but output an empty text response. However the same configuration works with discordSR v1

**Comment by Rei-x** [comment]
@lmmfranco 
See if #16 fixes your issue.
